services/models.py

```diff
@@ -666,6 +677,15 @@ def latitude(self, value):
             self.location = Point(0, 0)
         self.location[1] = value
 
+    def get_thumbnail_url(self, width=100, height=100):
+        """Shortcut to get the URL for an image thumbnail."""
+        if self.image and hasattr(self.image, 'url'):
+            frmt = "PNG" if self.image.path.rsplit(".", 1)[-1].upper() == "PNG" else "JPEG"
```

Comment:
IMHO, `if self.image.path.lower().endswith('.png')` would be easier to read here.